### ISSUE TITLE
Make it independent from swspotify.

### ISSUE THREAD

**Issue by xatuke** [SwagLyrics/SwagLyrics-Chrome-Extension#9]
# Why?
![img](https://imgur.com/UX5dZN9.png)
# Where?
![img](https://imgur.com/dei1qvu.png)
# How to fix senpai?
![img](https://imgur.com/Mx1zkcF.png)
# How actually?
### Write the same requests which are used for getting the lyrics on `swaglyrics cli` on the chrome extension, ez fix.

**Comment by aadibajpai** [comment]
@flabbet I know we should do this eventually but this also involves making a nice UI on the right and rewriting the swaglyrics core in JS.

thoughts?

**Comment by flabbet** [comment]
we could do that, but it's a bit of work and I don't have time and will to do that. Let's make it hacktoberfest thing and let people do that

**Comment by aadibajpai** [comment]
If you'd like to work on this, a UI like [Netflix Party](https://chrome.google.com/webstore/detail/netflix-party/oocalimimngaihdkbihfgmpkcpnmlaoa?hl=en) would be very interesting. Even SwagLyrics on Firefox Side View looks pretty nice w/o the extra scrollbars and stuff. 

![image](https://user-images.githubusercontent.com/27063113/92325823-261fb300-f06b-11ea-8a0e-33abc72b0f8b.png)

**Comment by flabbet** [comment]
Sure, but this is not what extension is for

**Comment by flabbet** [comment]
The extension is a bridge for SwagLyrics-For-Spotify, so it can support the Spotify browser. This is not a good idea to make console lyrics.

**Comment by aadibajpai** [comment]
I agree with @flabbet here, @crowded-geek. Your issue is actually two separate things, 

bug: the chrome extension spams local requests which floods network logs

feature request: standalone version of the chrome extension

now this isn't the same thing since SwSpotify is consumed by other python libraries as well which might wish to use the extension to extend support for Spotify Web. 

so even if we implement a standalone version, I don't think we can stop the bug.

**Comment by xatuke** [comment]
The fact still stays that the chrome extension doesn't need to use SwSpotify. I have no idea why was it connected to it in the first place. We can just scrape the current song data from the spotify tab, get lyrics and show them.

**Comment by aadibajpai** [comment]
> The fact still stays that the chrome extension doesn't need to use SwSpotify. I have no idea why was it connected to it in the first place. We can just scrape the current song data from the spotify tab, get lyrics and show them.

extension doesn't need SwSpotify but SwSpotify needs the chrome extension, it's meant to extend support for SwSpotify.

For example, suppose you create a python application that interacts with Spotify and you use SwSpotify, now how do you make it support the Spotify Web Player w/o the Chrome Extension?
